dali/sasha/saarch.cpp

```diff
@@ -1007,11 +1007,12 @@ class CWorkUnitArchiver: public CBranchArchiver
             }
             getWorkUnitCreateTime(wuid,time);
             // get latest time stamp
-            Owned<IPropertyTreeIterator> iter = e.getElements("TimeStamps/TimeStamp/*");
+            Owned<IPropertyTreeIterator> iter = e.getElements("Statistics/Statistic");
             if (iter) {
                 ForEach(*iter) {
                     CDateTime cmp;
-                    cmp.setString(iter->query().queryProp(NULL));
+                    timestamp_type ts = iter->query().getPropInt64("@ts");
```

Comment:
@jakesmith This is slightly different to before.  I think before it would have been the most recent timestamp.  Now it will be the most recent stats - both timestamps and other.  Could this cause any problems?  (I can't think of any at the moment.)

Reply:
I think it's ok, arguably an improvement, if a stat. outside of a TimeStamp updates, it would be better to consider it 'touched'.
That may changed conceivably, if we added stats for things like how many times the workunit had been looked at or something I suppose.